src/components/CategoryItem/CategoryItem.jsx

```diff
@@ -0,0 +1,24 @@
+import { useNavigate } from "react-router-dom";
+import { useFilters } from "../../contexts/product-context";
+export const CategoryItem = ({ category }) => {
+  const { dispatchFilters } = useFilters();
+  const navigate = useNavigate();
+  const { imgSrc, categoryName, categoryIdentifier } = category;
+  return (
+    <div
+      onClick={() => {
+        dispatchFilters({ type: "CLEAR_ALL" });
+        dispatchFilters({ type: "CATEGORY", payload: categoryIdentifier });
+        navigate("/products");
+      }}
+      className="product-category flex-col-center"
+    >
+      <img
+        className="category-img responsive-img"
+        src={imgSrc}
+        alt={categoryName}
+      />
+      <h3>{categoryName}</h3>
+    </div>
+  );
```

Comment:
This was the main learning part for me of how to redirect from the home page to products page with the selected category by running the dispatch functions related to clearing and then applying the category Image and the navigate function to redirect to the products